src/state/slices/txHistorySlice/txHistorySlice.test.ts

```diff
@@ -191,16 +186,20 @@ describe('txHistorySlice', () => {
         }),
       )
 
-      expect(store.getState().txHistory.txs.byAccountId[ethAccountId]).toStrictEqual([
+      expect(
+        store.getState().txHistory.txs.byAccountIdAssetId?.[ethAccountId]?.[ethAssetId],
```

Comment:
According to our types (and implementation) the `byAccountIdAssetId` object should always exist.

```suggestion
        store.getState().txHistory.txs.byAccountIdAssetId[ethAccountId]?.[ethAssetId],
```